setup/setup.md

```diff
@@ -1,10 +1,14 @@
 # Setup
 
-Choose your operating system and follow the instructions outlined.
+Choose your operating system and follow the instructions outlined. Each document will walk you through setup for your OS. Below is a brief description of the tools you will be installing and why you need them.
 
-Setup Checklist For All Operating Systems:
+## Tools required to complete course exercises
 
-* **Text editor.** We recommend [Atom](https://atom.io/).
-* **Scala.** Follow the directions for your operating system.
-* **Doodle.** Follow the directions for your operating system.
+* *[Java](https://java.com/en/)* - required to run Scala programs. Your java installation includes the Java Virtual Machine (or JVM), which provides the runtime environment for Scala.
+* *[Doodle](https://github.com/underscoreio/doodle)* - this is a Scala project that includes all the dependencies you will need to complete course exercises, including both Scala and sbt. This is why you don't need to install Scala or sbt separately in order to complete the exercises.
```

Comment:
I thought I'd include them for completeness. It does say above to use the page for your OS. If you feel strongly I can remove them.